src/darwin/Framework/CHIP/MTRDeviceControllerFactory.mm

```diff
@@ -1188,7 +1203,7 @@ - (instancetype)init
     _productAttestationAuthorityCertificates = nil;
     _certificationDeclarationCertificates = nil;
     _port = nil;
-    _shouldStartServer = NO;
+    _shouldStartServer = YES;
```

Comment:
Is this a change in default behavior?

Reply:
Yes, but for an init method that has never shipped.

Basically, if you use per-controller storage the default is that we start a CASE server, but by default we do not advertise any of the controllers' operational identities.

Unfortunately, the core code doesn't support lazily starting up the CASE server when the first controller that does want to advertise is detected.  But we could work on adding something to do that as a prereq for this....